add_bookmarks.py

```diff
@@ -53,7 +53,7 @@ def crawl_tree(tree, parent):
         pdf_out_filename = name_parts[0] + '(new)' + name_parts[1]
 
     # save `pdf_out`
-    outputStream = file(pdf_out_filename, 'wb')
+    outputStream = open(pdf_out_filename, 'wb')
     pdf_out.write(outputStream)
     outputStream.close()
```

Comment:
This code was written long ago, maybe it's better to refactor the above three lines as follows:
```python
with open(pdf_out_filename, 'wb') as output_stream:
    pdf_out.write(output_stream)
```